MinMaxGame.java

```diff
@@ -1,99 +1,99 @@
 import java.util.Scanner;
 
 class MinMaxGame extends Game {
+	// Implements the negamax tree search algorithm.
 
 	public MinMaxGame(int height, int width, boolean playersTurn) {
 		super(height, width, playersTurn);
 	}
-	@override
+	@Override
 	protected int computerMoves() {
 		/* Computer moves. Afterwards returns
 		0 : Nobody won yet
 		2 : Computer won
 		3 : Table is full  */
 
-		int moveY=0;
-		int moveX=0;
-		int[] bestMoveValue = new int[] {-10, 0};
-		int[] moveValue = new int[2];
+		if (movesCompleted == 0)
+			printTable();
 
-		for(int i = 0; i < height; i++) {
-			for(int j = 0; j < width; j++) {
-				if (table[i][j] == 0) {
-					// Trying a move, getting its value for the enemy and the
-					// length of the resulting game assuming optimal play.
-					table[i][j] = 2;
-					moveValue = getValue(table, i, j, 1, 1);
-					table[i][j] = 0;
+		System.out.println("Thinking...");
```

Comment:
Is this needed?

Reply:
Yes! So that user does not think the game has frozen.